### PR TITLE
Fix rename proto errors

### PR DESCRIPTION
**Pull request by alexshtin**
**What changed?**
Few fields were incorrectly renamed. This PR fix those errors. And also few google api-linter errors.

**Why?**
Errors must be fixed!

**How did you test it?**
Run all tests.

**Potential risks**
No risk.

### CODE REVIEW

File: client/clientBean_mock.go [hunk 1]
```diff
@@ -29,12 +29,13 @@
 package client
 
 import (
+	reflect "reflect"
```
Comment (mfateev) [inline client/clientBean_mock.go:32]: I don't understand this

File: common/archiver/gcloud/README.md [hunk 2]
```diff
@@ -1,5 +1,5 @@
 # Google Storage blobstore
-## Configuration
+## Config
```
Comment (mfateev) [inline common/archiver/gcloud/README.md:2]: Revert

File: common/archiver/s3store/README.md [hunk 3]
```diff
@@ -1,5 +1,5 @@
 # Amazon S3 blobstore
-## Configuration
+## Config
```
Comment (mfateev) [inline common/archiver/s3store/README.md:2]: revert

File: client/clientBean_mock.go (hunk 1 above)
Comment (alexshtin) [reply client/clientBean_mock.go:32]: It's ok, nobody does :-)

**Pull request merged**